### ISSUE TITLE
"content servers unreachable" after force enabling SteamPlay for a native game.

### ISSUE THREAD

**Issue by DPTJKKVH** [ValveSoftware/steam-for-linux#6033]
#### Your system information

* Steam client version (build number or date): Jan 17 2019
* Distribution (e.g. Ubuntu): Ubuntu 18.10 64bit
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
After enabling SteamPlay for a native Game (Dungeon Defenders / AppID: 65800) the game queued and update, probably trying to download the windows build of the game. However downloading fails immediately with the following error: "An error occurred while updating Dungeon Defenders (content servers unreachable)." This seems to have killed all downloads for me because disabling SteamPlay will still produce the same error message.

2 Things should be noted:

1. I downloaded some updates just fine right before forcing SteamPlay on Dungeon Defenders. So there is not issue with my network connection.

2. Some folks over at GamingOnLinux say that this issue happens only when IPv6 is not available and can be fixed by enabling IPv6. See [here](https://www.gamingonlinux.com/articles/another-steam-client-beta-is-out-adds-the-ability-to-force-steam-play.13376/page=7#r146104) and [here](https://www.gamingonlinux.com/articles/another-steam-client-beta-is-out-adds-the-ability-to-force-steam-play.13376/page=8#r146109). I cannot test this because I simply do not have an IPv6 connection from my provider and can only access the internet via IPv4. (Also I don't think Valve is intending to force IPv6 on anyone.)

#### Steps for reproducing this issue:

1. Opt-in to Steam Beta
2. Force SteamPlay for a native Linux ported game
3. Try to update or start the game to trigger download of the windows build.
4. Receive error message.

![error_message](https://user-images.githubusercontent.com/7238653/51408720-0408b280-1b60-11e9-986b-12ceb73f31b9.png)

**Comment by DPTJKKVH** [comment]
@Plagman there are huge indications that the beta build somehow makes content servers only reachable via IPv6 which is probably (hopefully) an accident. Would be great if you could look into that first to possibly safe you and everyone else a lot of time. :)

**Comment by kisak-valve** [comment]
Hello @DPTJKKVH, how you've written this report implies that there's a difference between native and proton games versus all games being affected with the current Steam client beta. If native games are equally affected, we should close this issue report as a duplicate of #6028.

**Comment by DPTJKKVH** [comment]
@kisak-valve after some testing I'm positive that this is rather a beta client issue and not an issue with the new SteamPlay. With the beta client active all download of all games (native or not) are impossible. I was just confused by the fact that I noticed this issue only after force enabling SteamPlay on a game.

So yes this is a dupe of #6028. I'll close it for you.

I will move my IPv6 information to #6028 so it does not get lost.
